__device-tests__/gutenberg-editor-sanity-test-1-visual.test.js

```diff
@@ -249,9 +252,7 @@ describe( 'Gutenberg Editor - Test Suite 1', () => {
 			// Wait for the modal to open
 			await editorPage.driver.sleep( 3000 );
 
-			const cellId = isAndroid()
-				? 'Column 1. Width is 50 Percent (%)., double-tap to change unit'
-				: 'Column 1. Width is 50 Percent (%).';
+			const cellId = 'Column 1. Width is 50 Percent (%).';
```

Comment:
Similar to [this change](https://github.com/WordPress/gutenberg/pull/51303/commits/9b8d7be878c557e7dee57ce81701c88da62cc64c), the accessibility hint is no longer appended to the accessibility content description.